doc/examples.xml

```diff
@@ -309,3 +309,49 @@ gap> GeneralisedPetersenGraph(IsMutableDigraph, 9, 4);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="HaarGraph">
+<ManSection>
+  <Oper Name="HaarGraph" Arg="[filt, ]n"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    If <A>n</A> is a positive integer then this operation returns
+    the <E>Haar graph</E> <M>H(<A>n</A>)</M>. <P/>
+
+    From
+    <URL>https://mathworld.wolfram.com/HaarGraph.html</URL>:
+    <P/>
+
+    <Q>A Haar graph is a bipartite regular graph indexed by a positive integer.
+    The number of vertices in the resulting graph is equal to the number of digits 
+    required to represent the indexing integer in binary. The vertices are 
+    then split into two groups and connected based on the binary representation
+    of the indexing integer. The two groups can be thought of as two vertical 
+    columns opposite each other. To check which vertices are connected, each vertex in
+    the first collumn is checked in turn. For each of these vertices the binary 
+    representation is mapped onto the second collumn, the 1s place coinciding with 
+    the vertex directly opposite the original vertex. An edge is added from the vertex
+    being checked to each vertex that is mapped to a 1 in the binary representation.</Q>
```

Comment:
I think this isn't totally clear, and I would probably replace it with something like 
```
The <M>i</M>th vertex in the first column is connected to the <M>j</M>th vertex in the second column if the binary representation of <A>n</A> has a 1 in position (<M>j-i</M> modulo <M>m</M>) + 1 from the left, where <M>m</M is the number of digits in the binary representation.
```